src/util/objPath.ts

```diff
@@ -1,15 +1,41 @@
+import _ from "lodash"
 import * as objectPath from "object-path"
 
 function normalisePath(path: string[]): string[] {
     return path.map((p) => String(p).trim()).filter((p) => p.length > 0)
 }
 
-export function objGet<T>(
+export function objGet<T extends object>(
     object: { [key: string]: T },
     path: string[],
-    defaultValue?: T,
+    defaultValue?: T | null,
 ): T | undefined {
-    return objectPath.get(object, normalisePath(path), defaultValue)
+    const result: T | undefined | null = objectPath.get(
+        object,
+        normalisePath(path),
+        defaultValue,
+    )
+
+    // Remove subcollections
+    if (result) {
+        const toOmit = Object.keys(result)
+            .map((key) => {
+                const innerValue: any = (result as any)[key]
+                return (
+                    (innerValue &&
+                        innerValue._meta && { key, remove: true }) || {
+                        key,
+                        remove: false,
+                    }
+                )
+            })
+            .filter((res) => res.remove)
+            .map((res) => res.key)
```

Comment:
There's already a helper function to do this:

https://github.com/freetrade-io/ts-firebase-driver-testing/blob/4b03b766375b73bb17d91b390b2f99ae59b96b07/src/util/stripMeta.ts#L3

Reply:
Yeah, I was trying to use that but reverted back to my partial working version